cpp-client/tools/CMakeLists.txt

```diff
@@ -70,6 +70,9 @@ target_link_libraries(nvidiaAIAASession NvidiaAIAAClient ${CMAKE_DL_LIBS})
 add_executable(nvidiaAIAADeepgrow aiaa/aiaa-deepgrow.cpp)
 target_link_libraries(nvidiaAIAADeepgrow NvidiaAIAAClient ${CMAKE_DL_LIBS})
 
+add_executable(nvidiaAIAAInference aiaa/aiaa-inference.cpp)
```

Comment:
This file should be checked-in together in this PR?

Reply:
yes.. this will build and install the example aiaa-inference.cpp

Reply:
yes.. checked-in